src/lib/kdf/xmd/info.txt

```diff
@@ -0,0 +1,11 @@
+<defines>
+XMD -> 20240404
+</defines>
+
+<module_info>
+name -> "XMD"
```

Comment:
As far as I can tell, the module exposes an internal API, only. Perhaps, mark it as `type -> "Internal"`?

Reply:
Right thanks